db/migrate/20220920153113_create_posts.rb

```diff
@@ -0,0 +1,12 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.string :title
+      t.string :text
```

Comment:
- [ ] Merely adding this text here does not add the `text` property to your Post table. You need to generate another migration file for adding a column to the posts table and run `db:migrate` again. If you look at your schema.rb file you would see that the text property is not in the posts table.

  You can take a look at this answer on how to generate migrations in rails: [https://stackoverflow.com/a/4834906](https://stackoverflow.com/a/4834906)